### ISSUE TITLE
fix: conditional files cannot escape the variable

### ISSUE THREAD

**Issue by mrgnhnt96** [felangel/mason#790]
**Description**

If a file is wrapped with a conditional variable `{{#var}}...{{/var}}` and the file name is an _**escaped**_ variable `{{{name}}}.dart`. The file name does not get resolved.

**Steps To Reproduce**

Create a `__brick__` folder with this file structure

```ascii
.
├── {{#is_plural}}{{{name}}}s_bloc.dart{{
│   └── is_plural}}
├── {{#is_plural}}{{{name}}}s_event.dart{{
│   └── is_plural}}
└── {{#is_plural}}{{{name}}}s_state.dart{{
    └── is_plural}}
```

the generated files will resolve as

```ascii
.
├── s_bloc.dart
├── s_event.dart
└── s_state.dart
```

**Expected Behavior**

I would expect the generated files to resolve as

```ascii
.
├── models_bloc.dart
├── models_event.dart
└── models_state.dart
```

**Additional Context**

A current workout around is to not escape the variable

```ascii
.
├── {{#is_plural}}{{name}}s_bloc.dart{{
│   └── is_plural}}
├── {{#is_plural}}{{name}}s_event.dart{{
│   └── is_plural}}
└── {{#is_plural}}{{name}}s_state.dart{{
    └── is_plural}}
```

**Comment by mrgnhnt96** [comment]
This also applies to directory names btw

**Comment by mrgnhnt96** [comment]
Fixed with #792
